### ISSUE TITLE
Compilation error for data type conversion.

### ISSUE THREAD

**Issue by technicalReport** [atrbx5/MotionDetectionCamera#2]
![error](https://user-images.githubusercontent.com/15214127/34368744-3750c772-eaf1-11e7-82ec-7895a897f5b9.jpg)

**Comment by atrbx5** [comment]
Do you have it on OSX 10.13 ?
I don't see anything like that

**Comment by technicalReport** [comment]
Yes, my mac os verison is 10.13.1. Can you show us how to get pose_iter_440000a.mlmodel? Thanks!

**Comment by atrbx5** [comment]
(you can use the one from the link in readme)
or
specify input resolution (originaly it was 1)

input: "image"
input_dim: 184 # Original: 368
input_dim: 184 # Original: 368

[convert.py.zip](https://github.com/atrbx5/MotionDetectionCamera/files/1588934/convert.py.zip)

**Comment by technicalReport** [comment]
![img_5174](https://user-images.githubusercontent.com/15214127/34402802-126cc30e-ebdf-11e7-898f-e91d5a44c231.PNG)
Thanks for your share. Attached image is my testing result for IOS version. The keypoint seems not accurate.  Can you tell what's wrong with it?  Thanks!

**Comment by atrbx5** [comment]
hm did you convert the network or used the one I shared?
do you have same result for osx?
